core/intrinsics/intrinsics.odin

```diff
@@ -25,20 +25,20 @@ alloca             :: proc(size, align: int) -> [^]u8 ---
 cpu_relax          :: proc() ---
 read_cycle_counter :: proc() -> i64 ---
 
-count_ones           :: proc(x: $T) -> T where type_is_integer(T) || type_is_simd_vector(T) ---
-count_zeros          :: proc(x: $T) -> T where type_is_integer(T) || type_is_simd_vector(T) ---
-count_trailing_zeros :: proc(x: $T) -> T where type_is_integer(T) || type_is_simd_vector(T) ---
-count_leading_zeros  :: proc(x: $T) -> T where type_is_integer(T) || type_is_simd_vector(T) ---
-reverse_bits         :: proc(x: $T) -> T where type_is_integer(T) || type_is_simd_vector(T) ---
-byte_swap            :: proc(x: $T) -> T where type_is_integer(T) || type_is_float(T) ---
+count_ones           :: proc(x: $T) -> T /*where type_is_integer(T) || type_is_simd_vector(T)*/ ---
+count_zeros          :: proc(x: $T) -> T /*where type_is_integer(T) || type_is_simd_vector(T)*/ ---
+count_trailing_zeros :: proc(x: $T) -> T /*where type_is_integer(T) || type_is_simd_vector(T)*/ ---
+count_leading_zeros  :: proc(x: $T) -> T /*where type_is_integer(T) || type_is_simd_vector(T)*/ ---
+reverse_bits         :: proc(x: $T) -> T /*where type_is_integer(T) || type_is_simd_vector(T)*/ ---
+byte_swap            :: proc(x: $T) -> T /*where type_is_integer(T) || type_is_float(T)*/ ---
```

Comment:
This file isn't meant to be parsed whatsoever in general. It's purely for documentation and will contains syntax errors.

Reply:
oh shoot, i figured, i just didn't know if those syntax errors were actually used in the docs. I'll revert that and push only the other thing if the other change is ok